packages/app-desktop/ElectronAppWrapper.ts

```diff
@@ -355,7 +384,14 @@ export default class ElectronAppWrapper {
 		const alreadyRunning = this.ensureSingleInstance();
 		if (alreadyRunning) return;
 
-		this.createWindow();
+		// Loads the Splash Screen and createWindow is returned from the below function
+		this.splashScreen();
```

Comment:
const splash = this.splashScreen()
this.createWindow()
this.win_.on('ready-to-show', () => {
    splash.destroy();
	this.win_.show();
});

Reply:
Are you sure `ready-to-show` is always called? What if the event has already been dispatched by the time you add the event listener?